src/fixit/cli.py

```diff
@@ -119,11 +119,13 @@ def main(
 @main.command()
 @click.pass_context
 @click.option("--diff", "-d", is_flag=True, help="Show diff of suggested changes")
+@click.option("--metrics", "-m", is_flag=True, help="Print metrics of this run")
```

Comment:
I think this would be better as a shared argument to `main()` that gets added to the `Options` object, so that it doesn't need to be duplicated on each subcommand that would care about it. I'd probably also suggest `--print-metrics` as the flag name, and maybe skip the short flag since I don't expect this to be a common use case.